lib/engine/game/g_18_esp/game.rb

```diff
@@ -1143,6 +1143,20 @@ def start_merge(corporation, minor, keep_token)
 
           # close corp
           close_corporation(minor)
+
+          # delete mz slot if mz
+          fix_cheater_slot_madrid(minor.city) if minor.id == 'MZ'
+        end
+
+        def fix_cheater_slot_madrid(slot)
```

Comment:
Calling this variable `slot` is confusing. It is the index of a city that is being passed to this method.

But I don't think that passing the city index will be reliable. MZ has a city index of 2, which is correct when the Madrid tile is yellow or green, but could MZ close after Madrid has been upgraded to brown, and there is a single city on the tile?